src/main/kotlin/com.bugsnag.android.gradle/BugsnagPluginExtension.kt

```diff
@@ -25,6 +24,7 @@ open class BugsnagPluginExtension(objects: ObjectFactory) {
     var isUploadNdkMappings: Boolean? = null
     var isReportBuilds = true
     var isUploadDebugBuildMappings = false
+    var isAutoUpdateBuildUuid = true
```

Comment:
Any reason this shouldn't be `Property<Boolean>`? Then it could be zipped with the buildUuid provider property or otherwise piped in as a property to tasks

Reply:
Good point - this has been updated to `Property<Boolean>`.